web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/user-name.html.twig

```diff
@@ -1,23 +1,10 @@
 {#
 /**
  * @file
- * Default theme implementation for displaying a username.
+ * Template for displaying a username.
```

Comment:
Well, it actually displays a lot more. Maybe the comment should read “Template for displaying a user“ – or we could get rid of this template and the comment (cf. suggestion on inlining two templates).